arbor/morph/place_pwlin.cpp

```diff
@@ -250,4 +250,56 @@ std::pair<mlocation, double> place_pwlin::closest(double x, double y, double z)
     return {loc, mind};
 }
 
+std::pair<std::vector<mlocation>, double> place_pwlin::all_closest(double x, double y, double z) const {
```

Comment:
I think we could avoid some code duplication here by calling `all_closest()` from within `closest()` and returning the first element only. Maybe not quite as efficient in some cases, but I'd assume having multiple locations of equal distance is rare anyway.